js/w3c/headers.js

```diff
@@ -155,6 +156,18 @@ define(
                     ret += ", <span class='ed_mailto'><a class='u-email email' " + rm + " href='mailto:" + p.mailto + "'>" + p.mailto + "</a></span>";
                 }
                 if (p.note) ret += " (" + p.note + ")";
+                if (p.extras && p.extras.length !== 0) {
```

Comment:
Sorry, was on vacation. Will get back to this soon. Just FYI, I want to rewrite ReSpec as much as possible in the declarative style. Specially in v4. 

Sent from my iPhone

> On Sep 30, 2015, at 12:44 AM, Ivan Herman <notifications@github.com> wrote:
> 
> In js/w3c/headers.js:
> 
> > @@ -155,6 +156,18 @@ define(
> >                      ret += ", <span class='ed_mailto'><a class='u-email email' " + rm + " href='mailto:" + p.mailto + "'>" + p.mailto + "</a></span>";
> >                  }
> >                  if (p.note) ret += " (" + p.note + ")";
> > +                if (p.extras && p.extras.length !== 0) {
> @marcoscaceres I really prefer your functional approach for coding this thing. However... the reason I did it the way I did it was because I followed the same style as elsewhere in the same function. Much as it is nicer to do this in your functional style, others, who may want to do any change in the future, may be bothered by the fairly asymmetric style of coding in this place. Ie, I am happy to accept/make those changes, but I am not sure it is wise to push this at this point...
> 
> I did add a trim() to the name in any case.
> 
> —
> Reply to this email directly or view it on GitHub.
>